lib/ansible/modules/monitoring/zabbix/zabbix_user_group.py

```diff
@@ -0,0 +1,331 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, OVH SAS
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: zabbix_user_group
+short_description: Create/update/delete Zabbix user group
+description:
+    - Create/update/delete Zabbix user group.
+version_added: "2.10"
+author:
+    - Emmanuel Riviere (@emriver)
+requirements:
+    - "python >= 2.7"
+    - "zabbix-api >= 0.5.3"
+options:
+    name:
+        description:
+            - Name of the Zabbix user group
+        required: true
+        type: str
+    debug_mode:
+        description:
+            - Enable or disable debug mode.
+        required: false
+        type: bool
+        default: no
+    gui_access:
+        description:
+            - Frontend authentication method.
+        required: false
+        type: str
+        choices: [default,internal,ldap,disabled]
+        default: "default"
+    users_status:
+        description:
+            - Whether the group is enabled of disabled
+        required: false
+        type: str
+        choices: [enabled, disabled]
+        default: "enabled"
+    rights:
+        description:
+            - List of permissions associated to the group
+            - Each right must contain a host_group and a permission (denied,RO,RW)
+        required: false
+        type: list
+        suboptions:
+            host_group:
+                description:
+                    - Host group name
+                type: str
+                required: true
+            permission:
+                description:
+                    - Permission associated with that group
+                type: str
+                required: true
+                choices: [denied, RO, RW]
+    users:
+        description:
+            -List of user aliases to add in the group
+        required: false
+        type: list
+    state:
+        description:
+            - State of the group
+            - If C(present), the group will be created or updated if the group configuration is different
+            - If C(absent), the group will be deleted
+        required: false
+        type: str
+        choices: [present, absent]
+        default: present
+
+extends_documentation_fragment:
+    - zabbix
+'''
+
+EXAMPLES = '''
+---
+- name: Create default user groups
+  local_action:
+    module: zabbix_user_group
+    server_url: http://127.0.0.1
+    login_user: username
+    login_password: password
+    state: present
+    name: Test group
+    rights:
+        - host_group: Admin VMs
+          permission: RW
+    users:
+        - user1
+        - user2
+'''
+
+RETURN = '''
+---
+'''
+
+import traceback
```

Comment:
```suggestion
import atexit
import traceback
```
Please import atexit module